---
layout: page
permalink: /research/
title: Research
---

Research in modern biology relies on a host of experimental techniques which shed light into a variety of phenomena that come together to constitute this amazing thing we call “life”. These experiments generate massive amounts of data, most of which are in the form of strings or sequences. Making sense of these sequences requires, apart from a keen understanding of biology, the ability to employ a range of tools from the arsenal of computer and computational sciences. This includes techniques such as design of algorithms and data-structures, statistical modeling, mathematical optimization, machine learning, software programming, etc. Exploring this rapidly growing field of computational biology is what I love to do.

## Reseach Projects
Some of my recent projects are described [in the website of Bioinformatics Lab, DLSU](https://www.bioinfodlsu.com/projects).

## Software
You check out some of my software projects at [my Github page](https://github.com/anishmss)
and some at [the lab's Github page](https://github.com/bioinfodlsu).

## Publications
Anish M.S. Shrestha, Mark Edward M. Gonzales, Phoebe Clare L. Ong, Pierre Larmande, Hyun-Sook Lee, Ji-Ung Jeung, Ajay Kohli, Dmytro Chebotarov, Ramil P. Mauleon, Jae-Sung Lee, Kenneth L. McNally: RicePilaf: a post-GWAS/QTL dashboard to integrate pangenomic, coexpression, regulatory, epigenomic, ontology, pathway, and text-mining information to provide functional insights into rice QTLs and GWAS loci, GigaScience, GigaScience 13, giae013 2024. 

Mark Edward M. Gonzales, Jennifer C. Ureta, Anish M.S. Shrestha: Protein embeddings improve phage-host interaction prediction, PLoS ONE 18(7): e0289030.

Kyle Christian L. Santiago and Anish M.S. Shrestha: DNA-protein quasi-mapping for rapid differential gene expression analysis in non-model organisms, GIW XXXI/ISCB-Asia V, Dec 12 – 14, 2022, Tainan, Taiwan. (to appear in BMC Supplements (??))

Anish M.S. Shrestha, Joyce Emlyn B. Guiao , Kyle Christian R. Santiago: Assembly-free rapid differential gene expression analysis in non-model organisms using DNA-protein alignment. BMC Genomics 23, 97 (2022).

Jennifer Ureta and Anish M.S. Shrestha: Identifying drug-resistant tuberculosis from chest X-ray images using a simple convolutional neural network, Journal of Physics, Conference Series, Vol 2071, International Conference on Biomedical Engineering (ICoBE) 2021.

Anish M.S. Shrestha, Crissa Ann I. Lilagan, Joyce Emlyn B. Guiao, Maria Rowena R. Romana-Eguia, Ma. Carmen Ablan Lagman: Comparative transcriptome profiling of heat stress response of the mangrove crab Scylla serrata across different sites. BMC Genomics 22, 580, 2021.

Anish M.S. Shrestha, Naruki Yoshikawa, and Kiyoshi Asai: Combining probabilistic alignments with read pair information improves accuracy of split-alignments, Bioinformatics 2018

Martin C. Frith and Anish M.S. Shrestha: A Simplified Description of Child Tables for Sequence Similarity Search,IEEE/ACM Transactions on Computational Biology and Bioinformatics, pp(99):1-1 2018

Anish M.S. Shrestha, Martin C. Frith, Kiyoshi Asai, Hugues Richard: Jointly aligning a group of DNA reads improves accuracy of identifying large deletions, Nucleic Acids Research, 2017

Anish Man Singh Shrestha, Martin C. Frith, and Paul Horton: A Bioinformatician’s Guide to the Forefront of Suffix Array Construction Algorithms, Briefings in Bioinformatics 2014 Mar; 15(2): 138-54.

Anish Man Singh Shrestha and Martin C. Frith: An Approximate Bayesian Approach to Mapping Paired-End DNA Reads to a Reference Genome , Bioinformatics (29) 8, pp 965 – 972, 2013; doi: 10.1093/bioinformatics/btt073.

Anish Man Singh Shrestha, Satoshi Tayu, Shuichi Ueno: Bandwidth of Convex Bipartite Graphs and Related Graphs, Information Processing Letters, Vol.112, pp. 411 – 417, 2012.

Anish Man Singh Shrestha, Asahi Takaoka, Satoshi Tayu, Shuichi Ueno: On Two Problems of Nano-PLA Design ,IEICE Transaction on Information and Systems, Vol E94-D, No.1, pp 35-41, Jan 2011.

Anish Man Singh Shrestha, Satoshi Tayu, Shuichi Ueno: On Orthogonal Ray Graphs, Discrete Applied Mathematics, Vol. 158, pp. 1650-1659, 2010.

## Presentations, Talks, etc.
Franz Stewart V. Dizon, Jennifer Ureta, Anish M.S. Shrestha: Performance of various protein representations for predicting phage-host interaction, 24th Philippine Computing Science Congress 2024.

Anish M.S. Shrestha: Understanding life through enormous pile of sequence data, Philippine Society for Microbiology Cluster 1 Symposium 2024

Anish M.S. Shrestha: Grouped-alignment of RNA-seq reads for accurate prediction of splice sites, 16th International Conference on Molecular Systems Biology ICMSB 2019, Manila.

Anish M.S. Shrestha: Challenges in Sequence Analysis in the Era of Big Biological Data, Invited Talk, Philippine Computing Science Congress 2019, Manila.

Biological sequence analysis in the era of high-throughput sequencers, Bioinformatics Syposium, Philippine Genome Center 2019, Manila.

Anish M.S. Shrestha, Martin C. Frith, Kiyoshi Asai, Hugues Richard: Jointly aligning a group of DNA reads improves accuracy of identifying large deletions, HiTSeq 2018, Chicago

Naruki Yoshikawa, Kiyoshi Asai, Anish M.S. Shrestha: Combining probabilistic alignments with read pair information improves accuracy of split-alignments, Poster at the 16th Asia Pacific Bioinformatics Conference (APBC) 2018, Kanagawa

Hideaki Noshiro, Kiyoshi Asai, Anish M.S. Shrestha: ZBDD-based representation and mining of RNA secondary structure ensemble, Poster at the 39th Annual Meeting of the Molecular Biology Society of Japan, 2016, Yokohama

Anish Man Singh Shrestha, Martin C. Frith, Kiyoshi Asai, Hugues Richard: A New Framework for Identifying Genomic Structural Variants Using Joint Alignment of Reads, Statistical Methods for Post Genomic Data Analysis 2016, Lille.

Anish Man Singh Shrestha, Martin C. Frith, Kiyoshi Asai, Hugues Richard: Identifying structural variations by jointly aligning a group of reads to a reference genome, GIW/ISCB-Asia 2014, Tokyo.

Anish Man Singh Shrestha and Martin C. Frith: An Approximate Bayesian Approach to Mapping Paired-End DNA Reads to a Reference Genome, ISCB-Asia/SCCG 2012, Shenzhen.

Satoshi Tayu, Anish Man Singh Shrestha, Shuichi Ueno: On the Complexity of Energy-Aware Mapping for NoCs, IEICE Technical Report, Vol.111, No.465, pp.219-224, 2012, Niigata.

Anish Man Singh Shrestha, Satoshi Tayu, Shuichi Ueno: Bandwidth of Convex Bipartite Graphs and Related Graphs, 17th International Computing and Combinatorics Conference (COCOON), Dallas, Lecture Notes In Computer Science, vol. 6842, pp. 307 — 318, 2011,
*Extended abstract appears in the proceedings of the 7th Hungarian-Japanese Symposium on Discrete Mathematics and Its Applications, pp. 381 — 388, 2011, Kyoto.

Anish Man Singh Shrestha, Satoshi Tayu, Shuichi Ueno: A Necessary Condition for Orthogonal Ray Graphs, Proceedings of the 2010 IEICE General Conference, A-1-29, 2010, Sendai.

Anish Man Singh Shrestha, Satoshi Tayu and Shuichi Ueno. On Two-Directional Orthogonal Ray Graphs, Proceedings of the IEEE International Symposium on Circuits and Systems (ISCAS), pp.1807-1810, 2010, Paris.

<!-- <h2>Publications</h2>
<ul>
	<li>
		<b>"Paper title #1"</b><br>
		<i>List of authors</i><br>
		Conference, Year<br>
		<a href=""><div class="color-button">pdf</div></a><a href=""><div class="color-button">cite</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
	<li>
		<b>"Paper title #1"</b><br>
		<i>List of authors</i><br>
		Conference, Year<br>
		<a href=""><div class="color-button">pdf</div></a><a href=""><div class="color-button">cite</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
</ul>

<h2>Research Projects</h2>
<ul>
	<li>
		<b>Project title</b><br>
		University, Duration<br>
		<i>Other details such as advisor's name may go here</i><br>
		<a href=""><div class="color-button">report</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
	<li>
		<b>Project title</b><br>
		University, Duration<br>
		<i>Other details such as advisor's name may go here</i><br>
		<a href=""><div class="color-button">report</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
</ul>

<h2>Research Implementations</h2>
<ul>
	<li>
		<b>Title #1</b>: Brief description of this research implementation.<br>
		<a href=""><div class="color-button">paper</div></a><a href=""><div class="color-button">report</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
	<li>
		<b>Title #2</b>: Brief description of this research implementation.<br>
		<a href=""><div class="color-button">paper</div></a><a href=""><div class="color-button">report</div></a><a href=""><div class="color-button">code</div></a>
	</li><br>
</ul> -->
